trade-finance-manager-ui/server/routes/utilisation-reports/index.test.js

```diff
@@ -0,0 +1,19 @@
+import { get } from '../../test-mocks/router-mock';
+
+const { getUtilisationReports } = require('../../controllers/utilisation-reports');
+
+describe('routes - utilisation-reports', () => {
+  beforeEach(() => {
+    require('.'); // eslint-disable-line global-require
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should setup routes with controllers', () => {
+    expect(get).toHaveBeenCalledTimes(1);
+
+    expect(get).toHaveBeenCalledWith('/', getUtilisationReports);
+  });
+});
```

Comment:
Should we even bother adding this test? I know it is done in the other files but I don't see the point in adding them for our new stuff (other than to remain consistent, which I think isn't a good enough reason when the test is this redundant)